File: PYTHON/7_ReverseInteger.py
# Name: Soniya Parate
# Date: 21/10/22

number = int(input("Enter the integer number: "))  
  
# Initiate value to null  
revs_number = 0  
  
# reverse the integer number using the while loop  
  
while (number > 0):  
    # Logic  
    remainder = number % 10  
    revs_number = (revs_number * 10) + remainder  
    number = number // 10  
  
# Display the result  
print("The reverse number is : {}".format(revs_number))  

File: C++/7.Reverse_Integer.cpp
//Name:Ishita Agrawal
//Date:29/10/22
class Solution {
public:
    int reverse(int x) {
      int digit,ans=0;
        while(x!=0){
        digit=x % 10;
            if((ans>INT_MAX/10)||(ans<INT_MIN/10)){
                return 0;
            }
         ans=(ans*10) +digit;
            x=x/10;
        }
        return ans;
        
    }
};